### ISSUE TITLE
issue in one line..urgent ..please resolve

### ISSUE THREAD

**Issue by bplm** [lucidrains/vit-pytorch#41]
train_data = CatsDogsDataset(train_list, transform=train_transforms)
valid_data = CatsDogsDataset(valid_list, transform=test_transforms)
test_data = CatsDogsDataset(test_list, transform=test_transforms)

in this line transform=val_transforms shoulb be there for valid_data , Am I correct1???

**Comment by niclaswue** [comment]
If I'm not mistaken, the transforms for validation and test set are exactly the same. Also, they should be the same as the validation accuracy would otherwise be meaningless if the val transforms would differ from the test set transforms.
